lambda/README.md

```diff
@@ -0,0 +1,73 @@
+# Getting the history for different SIMs
+
+By using the following GET request;
+
+```
+https://api.sim-details.nordicsemi.cloud/2024-07-01/sim/<your SIM's ICCID>/historicalData/timeSpan=<timespan>
```

Comment:
```suggestion
https://api.sim-details.nordicsemi.cloud/2024-07-01/sim/<your SIM's ICCID>/history?timeSpan=<timespan>
```

`timeSpan` is not a resource, but a filter. So using a query string here is more appropriate.

Reply:
I have been looking into this today and I can only find a function where I add Resources to the api root. Am I missing a function that could be used to add a query string, or does that involve making a lot of changes in the stack and with how I define the api?

And to be fair I forgot to update this README and I'm currently using:
'https://api.sim-details.nordicsemi.cloud/2024-07-01/sim/<your SIM's ICCID>/historicalData/<timespan>'

What would be the best option between having 'timeSpan=<timespan>' or '<timespan>' in the end?

Reply:
But you should be able to add this resource `<root>/history` to the API which then maps to the lambda function. And in that lambda function you handle the queryString which you will receive in the event.